BrowserKit/Tests/WebEngineTests/Utilities/WebKitTestHelpers.swift

```diff
@@ -0,0 +1,50 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import WebKit
+import XCTest
+
+/// Provides real `WKFrameInfo` and `WKSecurityOrigin` objects for tests.
+/// These types can no longer be created or mocked safely, as WebKit now requires
+/// fully initialized instances. This helper loads a lightweight `WKWebView`
+/// navigation and captures the frame and origin values that WebKit supplies.
+final class WebKitTestHelpers {
+    final class Delegate: NSObject, WKNavigationDelegate {
```

Comment:
Maybe we can name this as `FakeWKNavigationDelegate` to following conventional testing naming